CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+# 0.8.2 (2019-04-11)
+
+## Migration Disclaimer
+
+If you have the following set in your M3Query or M3Coordinator config, you should remove it. This is due to a bug in the LRU conversion cache within M3Query that is fixed in this release (see below).
```

Comment:
will it cause issues if they do not?

Reply:
I'm asking because I want us to minimise the number of migration disclaimers we issue. For e.g. in this case if the process works fine, it'd be better to just mark a deprecating warning; i.e. something end users do not _have_ to act on. Migration disclaimers (at least the one we've issued in the past), users cannot ignore.

Reply:
Yeah it won't cause any issues (at least not right away). But if we add a `cache` section in the config at some point, it would break.

Reply:
Happy to make this a warning or something instead.

Reply:
I'd rather delete this, if it breaks in future we can deal with it then right?

Reply:
Cool, in that case let’s remove from the release notes. In the future we’ll deal with this by adding a log message in the code which detects the presence of the deprecated fields.

Reply:
Hm just ensure it doesn’t break once. I remember vaguely our config parser for m3db was set in strict mode and would barf at unknown fields. Not sure if query uses the same thing?

Reply:
It actually does break it:
```
unable to load config from /Users/benjaminraskin/gocode/src/github.com/m3db/m3/src/dbnode/config/m3dbnode-local-etcd.yml: yaml: unmarshal errors:
  line 29: field cache not found in type config.Configuration
```

Reply:
Having said that, I'm not sure how many people actually set this in their config.